src/js/utils/fit-to-bounds.js

```diff
@@ -0,0 +1,29 @@
+import { transform } from 'utils/css';
+import { Browser, OS } from 'utils/browser'; 
+
+export function fitVideoUsingTransforms() {
+    return Browser.ie || (OS.iOS && OS.version.major < 9) || Browser.androidNative;
+}
+
+export default function fitToBounds(_videotag, width, height, stretching, styles) {
```

Comment:
I'd suggest renaming this file to better describe the modules it exports (eg. `video-fit.js`). Also, remove `default` and do explicit imports where needed.